### ISSUE TITLE
Task "Allow Node Exporter port in SELinux on RedHat OS family" fails on centos 7

### ISSUE THREAD

**Issue by sgargel** [cloudalchemy/ansible-node-exporter#152]
**What happened?**
Role apply over a fresh centos7 vagrant box fails with:
```
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ImportError: No module named seobject
fatal: [centos7]: FAILED! => {"changed": false, "msg": "Failed to import the required Python library (policycoreutils-python) on localhost.localdomain's Python /usr/bin/python. Please read module documentation and install in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"}
```
**Did you expect to see some different?**

The role should install missing python dependecies (`policycoreutils-python`). Role works with no problem on cenots8.

**How to reproduce it (as minimally and precisely as possible)**:

**Environment**

* Role version:

    `commit: bde46a6273283b22d1ef7277ebbcd390b3680825`

* Ansible version information:
   
    ```
  ansible 2.9.7`
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.17 (default, Apr 15 2020, 17:20:14) [GCC 7.5.0]
  ```

**Comment by paulfantom** [comment]
What is reported in `ansible_facts.distribution_major_version`?

**Comment by sgargel** [comment]
Sorry I can't check by now but seems this already been fixed in master with PR #145 

I've to check why ansible-galaxy has installed an older revision..

**Comment by paulfantom** [comment]
That fix is only on master branch. Ansible-galaxy uses last released version (which currently doesn't have a fix).
